### PR TITLE
chore: use .asf.yaml for apache workflow rule & add RAT in action

### PR DESCRIPTION
**Pull request by imbajin**
TODO:
- [ ] enable the required CI before release
- [ ] using the latest commitID in CI for server
- [x] enable rebase option for co-coding
- [x] enable rat check in action 

Do in next release: (1.x)
- [ ] merge all loader/client/hubble/tools action files into one so that we could reuse the env (could refer the [skywalking-ci](https://github.com/apache/skywalking/blob/master/.github/workflows/skywalking.yaml))

### CODE REVIEW

**Comment by javeme** [comment]
@imbajin Is this PR useless? if so we can close it

**Comment by imbajin** [comment]
> @imbajin Is this PR useless? if so we can close it

depend on shall we add `required` in ci..

File: .asf.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    rebase: true
+  protected_branches:
+    master:
+      required_status_checks:
+        # strict means "Require branches to be up to date before merging".
+        strict: true
+        # contexts are the names of checks that must pass
+        contexts:
+          - hugegraph ci/build
+          - hugegraph-ci/build
```
Comment (javeme) [inline .asf.yaml:43]: TODO: make some CI required

**Review by javeme — COMMENTED** [review]
In view of the fact that it is currently impossible to enable required context since it may skip some components CI due to the trigger mechanism, we just add an issue to track it and close this PR?

**Comment by imbajin** [comment]
both are fine (sine it have no diff now)

File: .asf.yaml (hunk 1 above)
Comment (imbajin) [reply .asf.yaml:43]: enable it(the rule) after the package rename done

**Review by Linary — DISMISSED** [review]

**Comment by imbajin** [comment]
@JackyYangPassion review the config with license checker, I found some files are excluded:
image

wonder to know if this is generally allowed to be excluded by the Apache projects/community?

seems we should exclude them carefully~

**Review by javeme — DISMISSED** [review]

**Review by Linary — APPROVED** [review]

**Comment by imbajin** [comment]
cherry pick the check in #428 for hubble

**Review by javeme — APPROVED** [review]

**Pull request merged**